package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "sequelize": "^3.30.2",
     "sequelize-cli": "^2.5.1",
+    "split.js": "1.3.5",
```

Comment:
@jdaguil -- The [CSS for the splitter](https://github.com/nexB/aboutcode-manager/blob/21-resize-views/assets/css/main.css#L305-L313) currently retrieves the gutter icons from their locations in `node_modules/split.js/grips/`.  We could move these `.png` files (along with the `grips` folder in which they sit) to `assets/images/`, paralleling our earlier relocation of `split.js` to `assets/js/`, and modify the CSS accordingly.  I don't know whether the `node_modules/...` directory for `split.js` contains any other necessary files but I'll test that approach.

BTW, here's what the `split.js` directory looks like in `node_modules`:

![image](https://user-images.githubusercontent.com/11096678/28890092-d94cea28-777a-11e7-9abd-051b514fa869.png)